examples/rtp/rtp_credit_ach_fallback_test.go

```diff
@@ -0,0 +1,95 @@
+package rtp
+
+import (
+	"context"
+	"testing"
+
+	"github.com/moovfinancial/moov-go/pkg/moov"
+
+	"github.com/go-faker/faker/v4"
+	"github.com/stretchr/testify/require"
+)
+
+func TestRTPCreditACHFallbackExample(t *testing.T) {
+	// Step 1: create Moov client, loading credentials from environment variables
+	mc, err := moov.NewClient()
+	require.NoError(t, err)
+
+	// Create a new context or use an existing one
+	ctx := context.Background()
+
+	// Ping the server to check credentials
+	err = mc.Ping(ctx)
+	require.NoError(t, err)
+
+	// Account IDs used
+	sourceAccountID := "ebbf46c6-122a-4367-bc45-7dd555e1d3b9"
+
+	// Step 2: Get the source wallet for our RTP transfer
+	sourcePaymentMethods, err := mc.ListPaymentMethods(ctx, sourceAccountID, moov.WithPaymentMethodType("moov-wallet"))
+	require.NoError(t, err)
+	require.Len(t, sourcePaymentMethods, 1)
+
+	sourcePaymentMethod := sourcePaymentMethods[0]
+
+	// Step 3: Create account for the destination
+	account, _, err := mc.CreateAccount(ctx, moov.CreateAccount{
+		Type: moov.AccountType_Individual,
+		Profile: moov.CreateProfile{
+			Individual: &moov.CreateIndividualProfile{
+				Name: moov.Name{
+					FirstName: faker.FirstName(),
+					LastName:  faker.LastName(),
+				},
+				Email: faker.Email(),
+			},
+		},
+	})
+	require.NoError(t, err)
+
+	// Step 4: Add (link) a bank account
+	bankAccountPayload := moov.BankAccountRequest{
+		HolderName:    "Jules Jackson",
+		HolderType:    moov.HolderType_Individual,
+		AccountType:   moov.BankAccountType_Checking,
+		RoutingNumber: "111326233", // this is a real routing number, that does not support RTP.
+		AccountNumber: "123456789", // fake it great!
+	}
+	bankAccount, err := mc.CreateBankAccount(ctx, account.AccountID, moov.WithBankAccount(bankAccountPayload), moov.WaitForPaymentMethod())
+	require.NoError(t, err)
+
+	t.Logf("BankAccountID: %v", bankAccount.BankAccountID)
+
+	// Step 5: Attempt to find the rtp-credit payment method
+	destinationPaymentMethods, err := mc.ListPaymentMethods(ctx, account.AccountID, moov.WithPaymentMethodType("rtp-credit"))
+	require.NoError(t, err)
+	require.Equal(t, 0, len(destinationPaymentMethods))
+
+	// Step 6: Fallback to ACH same-day
+	destinationPaymentMethods, err = mc.ListPaymentMethods(ctx, account.AccountID, moov.WithPaymentMethodType("ach-credit-same-day"))
+	require.NoError(t, err)
+	require.Greater(t, len(destinationPaymentMethods), 0)
```

Comment:
It would probably be more realistic to ask for rtp-credit but then `if` check to realize we didn't get one. We'd then need to ask for same day. I say this because the example assumes we won't get an rtp-credit, when in real code we may.

Reply:
I considered that, but with all of the surrounding test assertions I wasn't sure. Good point!

Reply:
@adamdecaf let me know what you think of the addition

Reply:
Thanks for the change. Looks good to me

Reply:
Thanks, that looks good